src/components/logocards.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { ReactNode } from 'react'
+
+interface cardprops{
```

Comment:
use camel case for component file name and add it inside a folder as well 
and keep the style sheet with in that folder